main.py

```diff
@@ -443,7 +443,8 @@ def _exit():
             # before it starts the databases. This will mean the port will be
             # available before queries can be executed for clusters and for
             # the enterprise edition in stand-alone mode.
-            time.sleep(10)
+            if not neo4j_config.version.startswith("4"):
```

Comment:
```suggestion
            if int(neo4j_config.version.split(".", 1)[0]) >= 5:
```

Don't want this to stop working once we get to Neo4j version 40.0 :clown_face: